bin/pycbc_multi_inspiral

```diff
@@ -268,7 +268,7 @@ with ctx:
         'bank_chisq': float32,
         'bank_chisq_dof': int,
         'cont_chisq': float32,
-        'slide_id': int
```

Comment:
I would be very cautious in making this change! You will likely want to do things like `slide_id == specific_slide_id` in what follows, and `slide_shift == specific_slide_shift` is a "bad idea TM" (https://www.geeksforgeeks.org/problem-in-comparing-floating-point-numbers-and-how-to-compare-them-correctly/) ... While you can implement "approximate equality" it is slower and can be more ugly (although I think numpy has "almost equal" functions built in for this).

Reply:
@jakeb245 was there a specific problem you were trying to solve with this change?

Reply:
Previously there was no information about the slide shifts being written to the output files (the `slide_id` that was previously there was not written due to some hardcoding). I thought putting the slide shift associated with each trigger would be a good way to fix this. I eventually went with writing the `timeslides` column instead.

I think I will revert this back to `slide_id`, and make sure it gets written to the output file properly.